<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveStateImage"/>
    <addaction name="actionLoadStateImage"/>
    <addaction name="actionFileLoadBin"/>
    <addaction name="separator"/>
    <addaction name="actionFileScreenshot"/>
    <addaction name="separator"/>
    <addaction name="actionScriptRun"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuEmulator">
    <property name="title">
     <string>Emulator</string>
    </property>
    <addaction name="actionEmulatorRun"/>
    <addaction name="actionEmulatorReset"/>
    <addaction name="separator"/>
    <addaction name="actionSoundEnabled"/>
    <addaction name="separator"/>
    <addaction name="actionEmulatorScreen0"/>
    <addaction name="actionEmulatorScreen1"/>
    <addaction name="actionEmulatorScreen2"/>
    <addaction name="actionEmulatorScreen3"/>
    <addaction name="actionEmulatorScreen4"/>
    <addaction name="actionEmulatorScreen5"/>
   </widget>
   <widget class="QMenu" name="menuDrives">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Drives</string>
    </property>
    <addaction name="actionDrivesFloppy0"/>
    <addaction name="actionDrivesFloppy1"/>
    <addaction name="actionDrivesFloppy2"/>
    <addaction name="actionDrivesFloppy3"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionConfBK10Basic"/>
    <addaction name="actionConfBK10Focal"/>
    <addaction name="actionConfBK10Fdd"/>
    <addaction name="actionConfBK11"/>
    <addaction name="actionConfBK11Fdd"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionDebugConsoleView"/>
    <addaction name="separator"/>
    <addaction name="actionDebugDebugView"/>
    <addaction name="actionDebugDisasmView"/>
    <addaction name="actionDebugMemoryView"/>
    <addaction name="actionDebugTeletypeView"/>
    <addaction name="separator"/>
    <addaction name="actionDebugStepInto"/>
    <addaction name="actionDebugStepOver"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulator"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuDrives"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEmulatorRun"/>
   <addaction name="actionEmulatorReset"/>
   <addaction name="separator"/>
   <addaction name="actionDrivesFloppy0"/>
   <addaction name="actionDrivesFloppy1"/>
   <addaction name="actionDrivesFloppy2"/>
   <addaction name="actionDrivesFloppy3"/>
   <addaction name="separator"/>
   <addaction name="actionEmulatorColorScreen"/>
   <addaction name="separator"/>
   <addaction name="actionSoundEnabled"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEmulatorRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtBkBtl.qrc">
     <normaloff>:/images/iconRun.png</normaloff>:/images/iconRun.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionEmulatorReset">
   <property name="icon">
    <iconset resource="QtBkBtl.qrc">
     <normaloff>:/images/iconReset.png</normaloff>:/images/iconReset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDrivesFloppy0">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtBkBtl.qrc">
     <normaloff>:/images/iconFloppySlot.png</normaloff>
     <normalon>:/images/iconFloppy.png</normalon>:/images/iconFloppySlot.png</iconset>
   </property>
   <property name="text">
    <string>Floppy A:</string>
   </property>
   <property name="iconText">
    <string>A</string>
   </property>
  </action>
  <action name="actionDrivesFloppy1">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtBkBtl.qrc">
     <normaloff>:/images/iconFloppySlot.png</normaloff>
     <normalon>:/images/iconFloppy.png</normalon>:/images/iconFloppySlot.png</iconset>
   </property>
   <property name="text">
    <string>Floppy B:</string>
   </property>
   <property name="iconText">
    <string>B</string>
   </property>
  </action>
  <action name="actionDrivesFloppy2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtBkBtl.qrc">
     <normaloff>:/images/iconFloppySlot.png</normaloff>
     <normalon>:/images/iconFloppy.png</normalon>:/images/iconFloppySlot.png</iconset>
   </property>
   <property name="text">
    <string>Floppy C:</string>
   </property>
   <property name="iconText">
    <string>C</string>
   </property>
  </action>
  <action name="actionDrivesFloppy3">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtBkBtl.qrc">
     <normaloff>:/images/iconFloppySlot.png</normaloff>
     <normalon>:/images/iconFloppy.png</normalon>:/images/iconFloppySlot.png</iconset>
   </property>
   <property name="text">
    <string>Floppy D:</string>
   </property>
   <property name="iconText">
    <string>D</string>
   </property>
  </action>
  <action name="actionFileScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionConfBK10Basic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BK 0010.01 BASIC</string>
   </property>
  </action>
  <action name="actionConfBK10Focal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BK 0010.01 FOCAL</string>
   </property>
  </action>
  <action name="actionConfBK10Fdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BK 0010.01 FDD</string>
   </property>
  </action>
  <action name="actionConfBK11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BK 0011M</string>
   </property>
  </action>
  <action name="actionConfBK11Fdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BK 0011M FDD</string>
   </property>
  </action>
  <action name="actionEmulatorScreen0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>512 x 256 BW</string>
   </property>
   <property name="iconText">
    <string>Color</string>
   </property>
  </action>
  <action name="actionFileLoadBin">
   <property name="text">
    <string>Load BIN</string>
   </property>
  </action>
  <action name="actionDebugDebugView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Processor View</string>
   </property>
  </action>
  <action name="actionDebugDisasmView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disasm View</string>
   </property>
  </action>
  <action name="actionDebugMemoryView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Memory View</string>
   </property>
  </action>
  <action name="actionDebugConsoleView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug Views</string>
   </property>
  </action>
  <action name="actionDebugStepInto">
   <property name="text">
    <string>Step Into</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionDebugStepOver">
   <property name="text">
    <string>Step Over</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionDebugTeletypeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Teletype View</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionScriptRun">
   <property name="text">
    <string>Run Script...</string>
   </property>
  </action>
  <action name="actionEmulatorScreen1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>512 x 256 Color</string>
   </property>
  </action>
  <action name="actionEmulatorScreen2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>512 x 384 BW</string>
   </property>
  </action>
  <action name="actionEmulatorScreen3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>512 x 384 Color</string>
   </property>
  </action>
  <action name="actionEmulatorColorScreen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="toolTip">
    <string>Color / BW</string>
   </property>
  </action>
  <action name="actionSoundEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QtBkBtl.qrc">
     <normaloff>:/images/iconSound.png</normaloff>:/images/iconSound.png</iconset>
   </property>
   <property name="text">
    <string>Sound Enabled</string>
   </property>
   <property name="iconText">
    <string>Sound</string>
   </property>
  </action>
  <action name="actionSaveStateImage">
   <property name="text">
    <string>Save State...</string>
   </property>
  </action>
  <action name="actionLoadStateImage">
   <property name="text">
    <string>Load State...</string>
   </property>
  </action>
  <action name="actionEmulatorScreen4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>896 x 512 BW</string>
   </property>
  </action>
  <action name="actionEmulatorScreen5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>896 x 512 Color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtBkBtl.qrc"/>
 </resources>
 <connections/>
</ui>
